core/src/main/java/org/opensearch/sql/expression/operator/arthmetic/MathUtil.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.expression.operator.arthmetic;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class MathUtil {
+
+  private MathUtil() {
+    // Utility class
+  }
+
+  /**
+   * Truncates a double number to required decimal places.
+   *
+   * @param numberToTruncate number to be truncated
+   * @param numberOfDecimals required decimal places
+   * @return truncated number as double
+   */
+  public static double truncateDouble(double numberToTruncate, int numberOfDecimals) {
+    return new BigDecimal(String.valueOf(numberToTruncate)).setScale(numberOfDecimals,
```

Comment:
I'm not sure it worthwhile to have this class.  These are all single-line calls used a single time.  I would prefer to remove this file and include the code in the `MathematicalFunction.java` code.  

Like:
```
            FunctionDSL.nullMissingHandling(
                (x, y) -> new ExprLongValue(
                    new BigDecimal(String.valueOf(numberToTruncate)).setScale(numberOfDecimals, numberToTruncate > 0 ? RoundingMode.FLOOR : RoundingMode.CEILING).doubleValue()),
            LONG, INTEGER, INTEGER),
```